modules/platforms/python/pyignite/api/sql.py

```diff
@@ -141,16 +143,17 @@ def scan_cursor_get_page(
 
 
 def sql(
-    connection: 'Connection', cache: Union[str, int],
+    conn: 'Connection', cache: Union[str, int],
```

Comment:
Won't this re-name break a backward compatibility?

Reply:
No, these functions are unrelated to the user API, and these arguments are used only as positional ones.